docs/validators/migrating.md

```diff
@@ -1,75 +1,91 @@
+### Stopping both the validator and the newly synced full node
+In order to avoid as much side affects as possible we will be stopping both nodes.
 
 ```
 systemctl stop desmos
 ```
 
-Once you have done so, you need to back up the following data:
+### Backup the following data in the validator node
 
-1. The mnemonic phrase associated with your key.  
-   You should have written it down on a piece of paper when you first created the node.
-   
-2. The validator private key.  
+In order to properly migrate our validator node to another server we will need to backup the following data(s):
+
+1. The validator private key.
    This is located inside the `~/.desmos/config/priv_validator_key.json` file.
-   
-3. The validator consensus state.  
+
+2. The validator consensus state.
    This is located inside the `~/.desmos/data/priv_validator_state.json` file.
+3. If you keep your keys on the node make sure you have the mnemonic phrase associated with your key(s).
 
-:::tip Do not delete your old node yet   
-We highly suggest you to delete your old node instance once that the new node is running properly. This will allow you to recover any information if you forgot to do so.  
-:::
+<!-- :::warning Do not move them in to the new fullnode just yet
+Back them up somewhere save but don't
+::: -->
 
-## 2. Setup your new node
-### Setting up the full node
-Once you are ready to get your new node running, the first thing to do is setup your node as a full node. To do this,
-you can read the guide [here](../fullnode/setup.md).
 
-Once that is done, stop your new node from running by using
+<!-- In order to properly transfer your validator to another server, you first have to stop the running node. To do this, you can execute the following command:
 
 ```
 systemctl stop desmos
-```
+``` -->
+
+<!-- Once you have done so, you need to back up the following data: -->
 
-Now, transfer the following backed up files from the old node into the new one:
+
+:::tip Do not delete your old node yet
+We highly suggest you to delete your old node instance once that the new node is running properly. This will allow you to recover any information if you forgot to do so.
+:::
+
+### Migrating the data in to the fullnode
+With both nodes stopped we will be copying the backedup data in to the newly synced fullnode.
+
+:::warning Warning
+At this point, both nodes should not be running. This is to prevent any possible side effects such as double signing
+:::
+
+Transfer the following backed up files from the old node into the new one:
 
 - `~/.desmos/config/priv_validator_key.json`
 - `~/.desmos/data/priv_validator_state.json`
 
-### Restoring your key
-To restored your validator key, all you have to do is execute the following command:
 
-```
-desmos keys add <your_key_name> --recover
-```
+### Startup the new validator node
+Once you have moved your `priv_validator_key.json` and `priv_validator_state` into the newly synced fullnode it will be recognized as a validator node.
```

Comment:
```suggestion
Once you have moved your `priv_validator_key.json` and `priv_validator_state.json`to the newly synced fullnode, it will be recognized as the same validator node.
```